script/deploy/healthCheck.ts

```diff
@@ -1,6 +1,6 @@
 // @ts-nocheck
 import { consola } from 'consola'
-import { $, spinner } from 'zx'
+import { $ } from 'zx'
```

Comment:
do we need to add zx to our package.json?

Reply:
`zx` is already in our dependencies in `package.json`

`"zx": "^8.2.2"`

Reply:
feel free to resolve